etc/adminhtml/system/adyen_advanced_order_processing.xml

```diff
@@ -24,7 +24,7 @@
 -->
```

Comment:
I think the word advanced is also redundant in the name of this file. It can be renamed to adyen_magento_order_processing.xml